datasets/representative_time/OneWeekdayDayAndOneWeekendDayPerMonthByHour.csv

```diff
@@ -0,0 +1,2305 @@
+scenario,model_year,geography,subsector,metric,month,hour,is_weekday,value
+scenario1,2030,6037,subsector1,electricity,1,0,FALSE,0.593521098171237
```

Comment:
Should the geography include a leading zero?

Reply:
The actual values aren’t being used for mapping, so it doesn’t matter as of now. But, we should be consistent in case anyone ever tries to map them.